aries-site/src/examples/templates/empty-state/AccessDenied.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Button } from 'grommet';
+import { Lock } from 'grommet-icons';
+import { EmptyState } from 'aries-core';
+
+export const AccessDenied = () => {
+  return (
+    <EmptyState
+      title="Access denied"
+      description={`You do not have access to this resource. 
+    Please contact the administrator to request access.`}
+      icon={<Lock size="xlarge" />}
```

Comment:
See related comment in EmptyState.js -- just one less piece of written guidance for us/callers to have to remember.

Reply:
Done.